src/main.ts

```diff
@@ -133,22 +136,23 @@ export function wrap<T>(
   }
 
   if (
-    has(cloudFunction, '__trigger.httpsTrigger') &&
-    get(cloudFunction, '__trigger.labels.deployment-callable') !== 'true'
+    !!cloudFunction?.__trigger?.httpsTrigger &&
+    cloudFunction?.__trigger?.labels['deployment-callable'] !== 'true'
   ) {
     throw new Error(
       'Wrap function is only available for `onCall` HTTP functions, not `onRequest`.'
     );
   }
 
-  if (!has(cloudFunction, 'run')) {
+  if (!cloudFunction.run) {
     throw new Error(
       'This library can only be used with functions written with firebase-functions v1.0.0 and above'
     );
   }
 
   const isCallableFunction =
-    get(cloudFunction, '__trigger.labels.deployment-callable') === 'true';
+    cloudFunction?.__trigger?.labels &&
+    cloudFunction?.__trigger?.labels['deployment-callable'] === 'true';
```

Comment:
Here again you can use `?.[]` to do null coalescing in a dictionary access.

Reply:
!! Thanks!